doc/conf.py

```diff
@@ -30,7 +30,7 @@
 html_logo = "_static/logo_horizontal.png"
 html_favicon = "_static/favicon.ico"
 
-html_theme_options.update({
```

Comment:
The theme options dict was updated because it comes pre-configured from nbsite:

https://github.com/pyviz-dev/nbsite/blob/fe2cded1cfc3519b6c423cd55f52ebc044fabb91/nbsite/shared_conf.py#L96-L100

@philippjfr Did you overwrite it on purpose?

Reply:
Yes, the only option in the shared conf were `footer_items` and these look cleaner than the override in nbsite.